src/pages/verificationConsentPage/index.js

```diff
@@ -1,59 +1,60 @@
-import React, { useCallback, useState } from "react";
-import { toast } from "react-toastify";
-import { useHistory } from "react-router-dom";
+import React, { useCallback, useState } from 'react'
+import { toast } from 'react-toastify'
+import { useHistory } from 'react-router-dom'
 
 import {
   Container,
   Card,
   TextContainer,
   Text,
   LargeText,
-  ButtonWrapper
-} from "./styles.js";
+  ButtonWrapper,
+} from './styles.js'
 
-import SmallButton from "components/smallButton";
-import api from "api/index.js";
-import walletFormContainer from "stateContainers/walletFormContainer";
+import SmallButton from 'components/smallButton'
+import api from 'api/index.js'
+import walletFormContainer from 'stateContainers/walletFormContainer'
 
 const cardText = [
   {
     paraOne:
-      "In order to verify do you consent to us sharing the following data with the laboratory you indicated that completed this test.",
-    boldOne: "Your verified mobile number",
-    boldTwo: "Your test reference number",
+      'In order to verify do you consent to us sharing the following data with the laboratory you indicated that completed this test.',
+    boldOne: 'Your verified mobile number',
+    boldTwo: 'Your test reference number',
     paraTwo:
-      "On successful verification, your COVI-ID status will be updated to verified.",
-    largeText: "Do you consent to this verification?"
-  }
-];
+      'On successful verification, your COVI-ID status will be updated to verified.',
+    largeText: 'Do you consent to this verification?',
+  },
+]
 
 const VerificationConsent = ({ cancel }) => {
-  const [loading, setLoading] = useState(false);
-  const history = useHistory();
-  const { covidTest, walletId } = walletFormContainer.state;
+  const [loading, setLoading] = useState(false)
+  const history = useHistory()
+  const { covidTest, walletId } = walletFormContainer.state
 
   const submitNewTest = useCallback(
-    async value => {
+    async (value) => {
       if (!value) {
```

Comment:
not need to check for `!value` anymore. the value has to be sent to the backend regardless now. So remove the `if`